components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -16,6 +16,7 @@
     <!-- Product and plugin compatibility requirements -->
     <!-- https://plugins.jetbrains.com/docs/intellij/plugin-compatibility.html -->
     <depends>com.intellij.modules.platform</depends>
+    <depends optional="true" config-file="maven.xml">org.jetbrains.idea.maven</depends>
```

Comment:
Maybe it's not being imported as expected?
Or maybe we should load it only after the client connects...
Let me confirm it.

Reply:
Although it's correctly defined, it's not loading `GitpodMavenProjectManager`. I'm trying an alternative to confirming if it works: https://github.com/gitpod-io/gitpod/pull/14656/commits/3890f49d69a4a11edd8c51d88ffb3da93df06fdb

Reply:
It ran `GitpodMavenProjectManager` this time, but hasn't detected the Maven modules either.

<img width="1077" alt="image" src="https://user-images.githubusercontent.com/418083/201676150-4864b09e-c997-4d9d-925e-6f858b497894.png">
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/418083/201676417-54422916-1899-4869-8ff6-9f2908b76ef0.png">

Reply:
I'm resolving this discussion; with the latest changes, it's working as expected.